okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/OkhttpTraceInterceptor.java

```diff
@@ -37,6 +37,10 @@
 
     @Override
     public Response intercept(Chain chain) throws IOException {
+        Tags.AttemptSpan attemptSpanTag = chain.request().tag(Tags.AttemptSpan.class);
```

Comment:
either re-use this tag below (line 49) or inline this call in the if clause.

Reply:
Can't inline because the `if` clause checks if `attemptSpanTag` is `null`. I didn't reuse the tag below b/c that finally block doesn't get run immediately, and I wasn't sure if it's okay to reuse the tag or if it could become stale. I'll change it to just reuse the tag.